splinterd/src/daemon.rs

```diff
@@ -169,6 +169,7 @@ impl SplinterDaemon {
                                 })?,
                         )
                     }
+                    #[cfg(feature = "database")]
```

Comment:
Why is this one guarded by “database”?

Reply:
Because the only supported in "memory" implementation for admin service store is a sqlite database so it requires create_store_factory. Although looking at it closer the store factory should probably always been pulled in. ill update that.